sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/AzureMonitorTraceExporter.cs

```diff
@@ -39,15 +39,15 @@ internal AzureMonitorTraceExporter(AzureMonitorExporterOptions options, ITransmi
         public override ExportResult Export(in Batch<Activity> batch)
         {
             // Add export time interval to data sample
-            _storageTransmissionEvaluator?.UpdateExportInterval();
+            _storageTransmissionEvaluator.UpdateExportInterval();
 
             // Prevent Azure Monitor's HTTP operations from being instrumented.
             using var scope = SuppressInstrumentationScope.Begin();
 
             try
             {
                 // Get number ticks before export
-                long ticksBeforeExport =  Stopwatch.GetTimestamp();
+                long timeBeforeExportInMilliSeconds = _storageTransmissionEvaluator.Stopwatch.ElapsedMilliseconds;
```

Comment:
Do you think it would be a better idea to have `StopWatch` in `AzureMonitorTraceExporter` and pass only `ElapsedMilliseconds` to the methods in `StorageTransmissionEvaluator`?